backend/src/pong/game/entities/game.entity.ts

```diff
@@ -32,17 +33,16 @@ export class Game {
 	@UpdateDateColumn()
 	updated_at: Date;
 
-	// to be joined later on
-	// @ManyToOne(() => User,  user => user.id)
-	// @JoinColumn({})
-	// player_one: User;
-	@Column()
-	player_one: number;
-
-	// @ManyToOne(() => User, user => user.id)
-	// @JoinColumn({})
-	// player_two: User;
-	@Check(`"player_two" <> "player_one"`)
-	@Column()
-	player_two: number;
+	@ManyToOne(() => User, (user) => user.id, {
+		nullable: false,
+		createForeignKeyConstraints: false,
```

Comment:
Shouldn't createForeignKeyConstraints be on `true`?
Now you can create games with non-existent users. If you remove this line it'll give a foreign key constraint error.

Reply:
i'm doing some cleaning in another branch and created a test for this

```ts
		it('should not allow non existent user', async () => {
			await expect(
				service.save({
					player_one: relationTestUsersIds[0],
					player_two: 89745,
				}),
			).rejects.toThrow('FOREIGN KEY constraint failed');
		});
```

Reply:
Yip! Changed!